### DISCUSSION TITLE
Wayland support

### DISCUSSION THREAD

**Discussion by videbar** [tutao/tutanota#7822] [Q&A] [answered]
Does the linux tutanota app support Wayland? I've read a few issues (e.g., #7765, #3974 ) that seem to hint at the fact that you can run the app natively on Wayland using the arguments `--enable-features=UseOzonePlatform --ozone-platform=wayland`  (is this documented somewhere?), but this doesn't work for me,  after launching the Appimage with these arguments, I checked with xwininfo and I still get an XWayland window.

**Comment by charlag** [comment]
Hi, it depends on Electron (and those are Electron flags). There was a change at some point that made Wayland default (maybe only on Flatpak) but it got rolled back due to issues.

**Reply by videbar** [reply]
Hi, thanks for your answer. So, is Wayland supported by electron now? Even if it's not the default, can it be enabled?

**Reply by charlag** [reply, marked as answer]
I think you are doing the right thing with command line flags but these might be the wrong ones. I think `--ozone-platform-hint=auto` might be the right one

**Reply by videbar** [reply]
Yes, that works, thanks!
